components/ChangemakerCard/ChangemakerCard.tsx

```diff
@@ -97,13 +97,14 @@ export const ChangemakerCard = ({
                   size={2}
                   leading="tight"
                   align="center"
+                  color="white"
                   className={styles.heading}
                 >
                   {heading}
                 </Heading>
               )}
               {subheading && (
-                <Text variant="card" align="center" leading="display">{subheading}</Text>
+                <Text variant="card" align="center" leading="display" color="white">{subheading}</Text>
```

Comment:
Previously the text on the front of the changemaker cards inherited the text color of the parent container. This makes sure that when they're place in a section with different color text, the text on the cards stay white.